runhouse/servers/env_servlet.py

```diff
@@ -64,7 +61,11 @@ async def wrapper(*args, **kwargs):
 class EnvServlet:
     async def __init__(self, env_name: str, *args, **kwargs):
         self.env_name = env_name
+        if "logs_level" in kwargs.keys() and kwargs.get("logs_level"):
```

Comment:
i think we can simplify this block to remove the if statement and just do:
`logger.setLevel(kwargs.get("logs_level", DEFAULT_LOG_LEVEL))`